front-end/src/Components/DataEntry.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom';
+import { onChangeHandler } from './Fucntions';
+import { onClickHandler } from './Fucntions';
+
+export default function DataEntry() {
+    const [data, setData] = useState('');
+    const navigate = useNavigate();
+
+    return (
+        <div className='text-center'>
+            <form onSubmit={e => {
+                console.log("onc click was called in the DataEntry.js");
+                e.preventDefault();
+                onClickHandler('details', 'POST', navigate, data);
+            }}>
+                <label htmlFor="FirstName">FirstName:</label><br />
+                <input className='border-2 rounded-lg' type="text" id="FirstName" name="FirstName" onChange={event => onChangeHandler(event, data, setData)} /><br />
+                <label htmlFor="LastName">LastName:</label><br />
+                <input className='border-2 rounded-lg' type="text" id="LastName" name="LastName" onChange={event => onChangeHandler(event, data, setData)} /><br />
+                <label htmlFor="Age">Age:</label><br />
+                <input className='border-2 rounded-lg' type="number" id="Age" name="Age" onChange={event => onChangeHandler(event, data, setData)} /><br />
```

Comment:
Again Clean Code Issue this should be like this
```
<input className='border-2 rounded-lg' 
    type="number" 
    id="Age" 
    name="Age"  
    onChange={event => onChangeHandler(event, data, setData)}
 />
<br />
```

Reply:
completed